firmware/controllers/engine_controller.cpp

```diff
@@ -93,6 +93,7 @@
 #include "lcd_controller.h"
 #include "pin_repository.h"
 #endif /* EFI_PROD_CODE */
+#include "tachometer.h"
```

Comment:
Can you please give me permissions to edit content of PR?

https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Reply:
just checked and you should have access, it is checked Allow write for maintainers.